vendor/github.com/mozilla/mig-sandbox/sandbox_linux.go

```diff
@@ -1,3 +1,4 @@
+// +build linux
```

Comment:
There should be a new empty line. But it ain't critical because the filename ends with _linux